radarr/anime-radarr.yml

```diff
@@ -1,16 +1,67 @@
-# Updated: 2023-08-01
+###################################################################################################
+# Recyclarr Configuration Template: Anime Radarr                                                  #
+# Updated: 2023-08-13                                                                             #
+# Documentation: https://recyclarr.dev                                                            #
+# Note: If you are using multiple profiles in a single instance, please read the following        #
+# documentation about file merging:                                                               #
+# https://recyclarr.dev/wiki/yaml/config-examples/#merge-single-instance                          #
+###################################################################################################
+
+# Instance Definition
+# This section defines your instance type, name, and authentication
+# Documentation: https://recyclarr.dev/wiki/yaml/config-reference/#basic
 radarr:
-  anime-radarr:  # A custom name used to identify this particular instance.
```

Comment:
Good idea removing this comment. Now that all of the names match the files, I think customizing them is less important.